beacon-chain/db/kv/state_test.go

```diff
@@ -155,7 +182,7 @@ func TestState_CanSaveRetrieve(t *testing.T) {
 				savedSt, err := db.State(context.Background(), root)
 				require.NoError(t, err)
 
-				assert.DeepSSZEqual(t, st.ToProtoUnsafe(), savedSt.ToProtoUnsafe())
+				require.DeepEqual(t, st.ToProtoUnsafe(), savedSt.ToProtoUnsafe())
```

Comment:
Why is this changed?

Reply:
I fixed the library but forget to revert back the line. I'll fix it